tutorial/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.187.0/containers/alpine/.devcontainer/base.Dockerfile
+
+# [Choice] Alpine version: 3.13, 3.12, 3.11, 3.10
+ARG VARIANT="3.13"
+FROM mcr.microsoft.com/vscode/devcontainers/base:0-alpine-${VARIANT}
```

Comment:
We could also pull your Docker container instead making this Dockerfile trivial, but this allows people to change it and also adds some niceties from the MS version of the container.

Reply:
Good point, I would like to leave the options open here, so let's stick to that solution